index.js

```diff
@@ -49,7 +49,7 @@ async function run() {
 		const releaseNotes = await generateReleaseNotes({range, exclude, commitTemplate, releaseTemplate, dateFormat, reverseSort, skipOnEmpty});
 
 		// Skip creating release if no commits
-		if (releaseNotes === null) {
+		if (releaseNotes === undefined) {
```

Comment:
Can be
```suggestion
		if (!releaseNotes) {
```

Reply:
I think it's best to avoid checking falsiness because other values such as empty string would also pass that check and it's possible the user may specify an empty string as the template to make the body empty (I'm not sure why they'd be use using release-*with-changelog* at that point, but still :joy:)